### PR TITLE
Added Kinieta Animation Engine

### PR DESCRIPTION
**Pull request by mmick66**
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 3 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 4 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
400 | Dupe | https://github.com/rob-nash/Records 
638 | [301](https://httpstatuses.com/301) | https://github.com/lucdion/FlexLayout redirects to<br>https://github.com/layoutBox/FlexLayout
2769 | [302](https://httpstatuses.com/302) | http://us11.campaign-archive1.com/home/?u=cd1f3ed33c6527331d82107ba&id=532dc7fb64 redirects to<br>https://us1.campaign-archive.com/home/?u=cd1f3ed33c6527331d82107ba&id=532dc7fb64
2842 | Error | https://ioscompanies.info/about/welcome <br> Connection refused - connect(2) for "ioscompanies.info" port 443


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by lfarah** [comment]
@mmick66 please add ```:large_orange_diamond:``` in the end of the description

**Comment by lfarah** [comment]
@aaronjsutton removed?

**Comment by aaronjsutton** [comment]
GitHub is showing `unknown repository` in the PR for me?

**Comment by lfarah** [comment]
@aaronjsutton he must have deleted the branch, but it still should work.

**Comment by aaronjsutton** [comment]
Oh, that makes more sense. Thanks!

File: README.md [hunk 1]
```diff
@@ -1761,6 +1761,7 @@ Most of these are paid services, some have free tiers.
 * [Gemini](https://github.com/shoheiyokoyama/Gemini) - Gemini is rich scroll based animation framework for iOS, written in Swift :large_orange_diamond:
 * [WaterDrops](https://github.com/LeFal/WaterDrops) - Simple water drops animation for iOS in Swift :large_orange_diamond:
 * [ViewAnimator](https://github.com/marcosgriselli/ViewAnimator) - ViewAnimator brings your UI to life with just one line. 🔶
+* [Kinieta](https://github.com/mmick66/kinieta) - An Animation Engine for iOS with an Intuitive API and Readable Code.
```
Comment (dkhamsing) [inline README.md]: Please add 🔶

**Comment by dkhamsing** [comment]
Closing for lack of activity

**Pull request closed without merging**